sdk/core/core-auth/CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 # Release History
 
-## 1.3.0 (Unreleased)
+## 1.3.0 (2021-03-30)
 
-- Adds the `AzureNamedKeyCredential` class which supports credential rotation and a corresponding `NamedKeyCredential` interface to support the use of static string-based names and keys in Azure clients.
 - Adds the `isNamedKeyCredential` and `isSASCredential` typeguard functions similar to the existing `isTokenCredential`.
```

Comment:
Why is this not under "new features"?

Reply:
We havent used the ### New Features before in this changelog when we added the other credentials and so the question. I wouldnt mind if we did not make the distinction at all

Reply:
Oh, fair enough! I'll remove that section header.

Reply:
Wasn't the new section(header) meant for the new release(highlights) automation?

Reply:
Yes, but now I think we're stating none of this is worthy of being in the highlights (but are fine for the regular changelog).

Which I think is fine...event hubs and service bus will call out that they can use these credentials, which is what customers should care about.